### PR TITLE
Update cats.md

### PR DESCRIPTION
**Pull request by danieljperry**
Added lots of information about CATs. This is meant to appear on chialisp.com.

Two definite issues (possibly more):
1. Lipa's paper is missing. Where can this be found online?
2. The truths from cat_truths.clib appear to have been modified recently, so they might not match what is listed here. In particular, CAT mod_struct may or may not be valid.

### CODE REVIEW

**Review by Quexington — COMMENTED** [review]
Just a few comments but otherwise it looks good!

**Comment by hoffmang9** [comment]
Lipa's paper is here https://github.com/Chia-Network/vdf-competition/blob/main/classgroups.pdf

**Comment by hoffmang9** [comment]
In context, I'm worried I gave you the wrong paper.

**Comment by paulhainsworth-chia** [comment]
I think Dan is looking for Lipa's latest work on CAT1 standard.

**Comment by danieljperry** [comment]
Asked Matt Howard about the Lipa paper in a PM.

**Comment by danieljperry** [comment]
Lipa's paper doesn't exist yet, so I removed the link.

File: docs/puzzles/cats.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Chia Asset Tokens (CATs)
 ## Introduction to CATs
 
 **Chia Asset Tokens (CATs)** are fungible tokens that are issued from XCH.
-The CAT1 Standard is the first (and so far only) CAT Standard. More information on the naming conventions used in this document can be found [here](https://www.chia.net/2021/09/23/chia-token-standard-naming.en.html "Blog entry explaining CAT1 naming conventions").
+The CAT1 Standard is the first (and so far only) CAT Standard. CAT1 is a draft standard as of Nov 16, 2021. After a comment period from the Chia community, CAT1 will be finalized. This may include additional capabilities, and may result in some breaking changes to existing CATs. More information on the naming conventions used in this document can be found [here](https://www.chia.net/2021/09/23/chia-token-standard-naming.en.html "Blog entry explaining CAT1 naming conventions").
```
Comment (hoffmang9) [inline docs/puzzles/cats.md:9]: I'd suggest "and could result in some breaking changes"

**Review by hoffmang9 — APPROVED** [review]
lgtm

**Pull request merged**